### ISSUE TITLE
Suggestion: Clean template requires a little

### ISSUE THREAD

**Issue by Vadinci** [LuckyKat/bento-amd-tool#19]
For example, the bento-define-entity-module includes a bunch of GUI components/modules. Although it's not really a problem, they clutter up the file a little and are probably not used for the majority of entities (or only one of them is). 

Seeing as the purpose of this tool is to make requiring new modules a breeze, it seems unnecessary to have this many 'default' requirements :)

**Comment by HernanZh** [comment]
Agreed. Suggestions for default requires? Maybe only Bento and Utils.

**Comment by Vadinci** [comment]
Probably those two, plus whatever is used in that specific template (e.g. `entity` and `sprite` in the Entity template). I think the best/tidiest practice is to only require modules that are actually used, but any modules that are used very often might be big enough timesavers to include by default (Vector2 and maybe Tween might be good candidates).

**Comment by HernanZh** [comment]
Made a new "bento module" snippet that's completely empty. Not sure yet about other snippets. Opinions? --> https://github.com/LuckyKat/bento-amd-tool/commit/6f9e201dfb7dfefebf96aacf72f482039183f07f
